apps/aws-amplify/src/lib/isVaildUrl.ts

```diff
@@ -0,0 +1,11 @@
+export const isValidUrl = (urlString: string) => {
```

Comment:
Initially just using this method to verify with builtin `URL` constructor, can migrate to a more finite `REGEX` pattern if we think its necessary, these webhook url's get pretty dicy though so open to opinions here

Reply:
All webhooks should follow the pattern of `https://webhooks.amplify.{amazon region}.amazonaws.com/prod/webhooks{some id and token}` so we could potentially use a regex pattern on `webhooks.amplify` or some variation on that.

Reply:
OK so thinking of security,we'llneed to restrict what can be in amplifyWebhookUrl (similar to what cors-anywhere calls originWhitelist), there is a risk of abuse to our service with arbitrary Urls. SO this might need to live in a seperate PR depending on the Lambda Function URL implementation 🤔 I say we add this as is and then in a follow up rework this

Reply:
Going to add a ticket for more robust validation on this when we actually have the business logic connected for the button trigger in a follow up PR